diff --git a/node_modules/react-native-gifted-chat/lib/Composer.js b/node_modules/react-native-gifted-chat/lib/Composer.js
index 7bbf561..1070c46 100644
--- a/node_modules/react-native-gifted-chat/lib/Composer.js
+++ b/node_modules/react-native-gifted-chat/lib/Composer.js
@@ -50,7 +50,7 @@ export function Composer({ composerHeight = MIN_COMPOSER_HEIGHT, disableComposer
             styles.textInput,
             textInputStyle,
             {
-                height: composerHeight,
+                minHeight: MIN_COMPOSER_HEIGHT,
                 ...Platform.select({
                     web: {
                         outlineWidth: 0,
diff --git a/node_modules/react-native-gifted-chat/lib/MessageContainer.js b/node_modules/react-native-gifted-chat/lib/MessageContainer.js
index 6bdf6da..85c7fc7 100644
--- a/node_modules/react-native-gifted-chat/lib/MessageContainer.js
+++ b/node_modules/react-native-gifted-chat/lib/MessageContainer.js
@@ -88,6 +88,9 @@ export default class MessageContainer extends React.PureComponent {
             }
         };
         this.handleOnScroll = (event) => {
+            if (this.props.listViewProps.onScrollEvent){
+                this.props.listViewProps.onScrollEvent(event);
+            }
             const { nativeEvent: { contentOffset: { y: contentOffsetY }, contentSize: { height: contentSizeHeight }, layoutMeasurement: { height: layoutMeasurementHeight }, }, } = event;
             const { scrollToBottomOffset } = this.props;
             if (this.props.inverted) {
